### DISCUSSION TITLE
正则表达式，能提供 匹配内容 列表么？

### DISCUSSION THREAD

**Discussion by leonyff** [he3-app/he3-feedback#176] [Ideas]
分组 匹配内容，列出列表，方便后期操作！  
类似下面这样：
![image](https://github.com/he3-app/he3-feedback/assets/38803732/06ba06ac-2efe-4ab5-9e3d-8a3773c8d7bb)

**Comment by lyzhang1999** [comment]
感谢您的建议，我已经创建了 issues：https://github.com/he3-app/he3-feedback/issues/177，请关注进展。
